curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/store-multiple-values-in-one-variable-using-javascript-arrays.md

```diff
@@ -19,8 +19,6 @@ You start an array declaration with an opening square bracket, end it with a clo
 
 Modify the new array `myArray` so that it contains both a `string` and a `number` (in that order).
 
-**Hint**  
-Refer to the example code in the text editor if you get stuck.
 
```

Comment:
```suggestion
```

This'll leave an extra blank line between the text. Can we remove that? 🙂

Reply:
Sure, I will update it 👍